content/flux/v0.x/stdlib/types/_index.md

```diff
@@ -0,0 +1,28 @@
+---
+title: Flux types package
+list_title: types package
+description: >
+  The Flux `types` package provides functions for working with
+  [Flux basic types](/flux/v0.x/data/data-types/basic/).
+  Import the `types` package.
```

Comment:
is this last line supposed to be part of the description?

Reply:
Yeah, it is. This is a convention followed with all the other packages.